### ISSUE TITLE
validate on pretrained model get accuracy 0.5000

### ISSUE THREAD

**Issue by muyoucun** [HsuTzuJen/Face_Recognition_Practice_with_TF#4]
i totally followed your code and i got my model.
while i was training, the logs can show that i get accuracy like 0.99 when validating on lfw.
but after i finished training and just saved a model (such as ./ckpt/mobilefacenet_acc_0.99), when i try to restore this model and do its validation on lfw, i always get the same wrong result. i use function ver_test from verification.py

best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
best_threshold_index 0 0.5
[lfw]Accuracy-Flip: 0.5000+-0.00000

so i wonder why this happened because it make sense when i was training but it get wrong after i finished the code. thank you!

**Comment by muyoucun** [comment]
well i have solved this. 
i only saved tf.trainable_variables() so the parameters of bn_layer (moving_mean and moving_variance) will not saved into ckpt. as a result everytime i restore the model, moving_mean and moving_variance will be zero, so the accuracy on lfw will be 0.5000.

**Comment by Joll123** [comment]
@muyoucun   你好 你实现了多高的准确率在lfw上。谢谢 ！

**Comment by Joll123** [comment]
@muyoucun 你是怎么保存模型的额  我现在也遇到你的问题了  谢谢

**Comment by muyoucun** [comment]
@Joll123 
把滑动平均这些东西手动加到要保存的变量集合里去就好了
MobileFaceNet_vars = [v for v in tf.trainable_variables() if v.name.startswith('mobilefacenet')]
bn_moving_vars = [g for g in tf.global_variables() if 'moving_mean' in g.name]
bn_moving_vars += [g for g in tf.global_variables() if 'moving_variance' in g.name]
MobileFaceNet_vars += bn_moving_vars
saver = tf.train.Saver(MobileFaceNet_vars, max_to_keep=args.saver_maxkeep)

**Comment by Joll123** [comment]
@muyoucun  谢谢

**Comment by Joll123** [comment]
@muyoucun 我读取lfw图片测试这个模型  而不是用bin文件进行测试，为什么准确率只有83%。你有做过吗。谢谢！

**Comment by muyoucun** [comment]
@Joll123  我没有试过诶。你瞧瞧是不是rgb什么的问题，或者研究一下原来的测试代码？

**Comment by Joll123** [comment]
@muyoucun 谢谢你的帮助  我对bin文件不太了解在验证时为什么做下面的操作
![image](https://user-images.githubusercontent.com/43200569/48596717-3e55a800-e995-11e8-830f-79303bd09cf7.png)

**Comment by muyoucun** [comment]
@Joll123 
额我找了找，大概就是这个吧==
https://github.com/happynear/NormFace/blob/master/MirrorFace.md
他这里把两个embedding直接相加，还有的方法是两个128维的embedding拼接成256维的

**Comment by Joll123** [comment]
@muyoucun 谢谢
